tests/runtests.py

```diff
@@ -396,8 +413,9 @@ def paired_tests(paired_test, options, test_labels, parallel):
              'LiveServerTestCase) is expected to run from. The default value '
              'is localhost:8081-8179.')
     parser.add_argument(
-        '--selenium', action='store_true', dest='selenium', default=False,
-        help='Run only the Selenium tests (equivalent to "--tag selenium").')
+        '--selenium', dest='selenium', action=ActionSelenium,
+        help='A comma separated list of browsers to run the Selenium test against. '
+             'You can run only the Selenium tests by adding "--tag selenium".')
```

Comment:
My point is there's no more way of running the full test suite anymore (by using `DJANGO_SELENIUM_TESTS=1 ./runtests.py`) because no browsers are included by default like it used to be. Do you think this is acceptable?